tests/php/InlineCssTest.php

```
@@ -0,0 +1,566 @@
+		$cache = $this->make_cache();
+
+		$GLOBALS['wp_version'] = '6.8';
+		$this->assertSame( 20000, $this->invoke_private( $cache, 'get_styles_inline_limit' ) );
+
+		$GLOBALS['wp_version'] = '6.9';
+		$this->assertSame( 40000, $this->invoke_private( $cache, 'get_styles_inline_limit' ) );
+
+		// Without a known $wp_version the newest default is used.
+		unset( $GLOBALS['wp_version'] );
+		$this->assertSame( 40000, $this->invoke_private( $cache, 'get_styles_inline_limit' ) );
+	}
```

PHPMD (2.15.0)

[error] test_get_styles_inline_limit_is_version_aware accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)